test_e2e/dest/couchbase.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
```

Comment:
This file needs to be updated to be executable (`chmod +x test_e2e/dest/couchbase.sh`)

Reply:
added